proto/alarm/v1/config.proto

```diff
@@ -14,14 +14,20 @@ message ProvideAlarmConfiguration {
 }
 
 message AlarmConfiguration{
-    string info = 1;
-    string units = 2;
-    string exec_path = 3;
-    string recipient = 4;
-    string conf_source = 5;
-    uint64 update_every = 6;
+    string name = 1;
+    string chart = 2;
```

Comment:
Why do we need those two in the configuration? They are already included here https://github.com/netdata/aclk-schemas/pull/20/files#diff-0f2d7f09a7df02f109133b68107ae510fbe5ae5b56279e1ec9a60e1cd0be58b5R46-R47

Reply:
Yes, I know....

The thing I noticed is that we don't have let's say a "key" for the configuration, something to show what the configuration is a about... Could be useful in debugging too ?

Reply:
We have the config_hash on the `ProvideAlarmConfiguration` message and this should be enough right?

Reply:
I should change the names of those two, they won't be exactly the same as the items name and chart in the AlarmEvent...

From here: https://learn.netdata.cloud/docs/agent/health/reference name as I have it now will be the `alarm/template` field (one of those two) and chart will be the `on` field from the config. I think we should include them....